Document behavior of generating destination path when copying (#1498)
copier-org/copier: docs/generating.md

copier.run_copy("path/to/project/template", "path/to/destination")
```

> If `path/to/destination` doesn't exist, `copier` will both create and populate the directory with the generated files. Note that this will raise an error if the destination is read-only.

Reviewer: ```suggestion
If `path/to/destination` doesn't exist, Copier will create the directory and populate it with the generated files. If `path/to/destination` exists, it must be writable (not read-only).
```
